package.json

```diff
@@ -57,5 +57,8 @@
     "futures"
   ],
   "author": "Tilde, Inc. & Stefan Penner",
-  "license": "MIT"
+  "license": "MIT",
+  "engines": {
+    "node": "0.12.* || ^4.2 || 6.*"
```

Comment:
`npm` won't actually warn for dependencies. But we should carry it forward for `yarn` which will respect it.

`"0.12.* || 4.* || 6.* || 7.*"` is what we've started doing, though what we really mean is "latest of each branch."